index.d.ts

```diff
@@ -635,6 +635,11 @@ declare namespace plugins {
      * @default true
      */
     middleware?: boolean;
+
+    /**
+     * List of urls to which propagation headers should not be injected
+     */
+    propagationBlockList?: string | RegExp | ((url: string) => boolean) | (string | RegExp | ((url: string) => boolean))[];
```

Comment:
```suggestion
    propagationBlocklist?: string | RegExp | ((url: string) => boolean) | (string | RegExp | ((url: string) => boolean))[];
```

Reply:
Not sure why TypeScript tests are passing since this uses a different key 🤔 . What happens if you run `yarn type:test` locally?

Reply:
If I run `yarn type:test` locally I get the following output:

```
❯ yarn type:test
yarn run v1.22.10
$ cd docs && yarn && yarn test
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...

success Saved lockfile.
$ tsc -p . && tsc test
$ node test
DATADOG TRACER CONFIGURATION - {"date":"2021-09-01T23:07:12.045Z","os_name":"Darwin","os_version":"18.7.0","architecture":"x64","version":"2.0.0-pre","lang":"nodejs","lang_version":"12.22.5","enabled":true,"service":"docs","agent_url":"http://127.0.0.1:8126","agent_error":"Network error trying to reach the agent: connect ECONNREFUSED 127.0.0.1:8126","debug":false,"sample_rate":1,"sampling_rules":[],"tags":{"service":"docs","version":"1.0.0","runtime-id":"19b9f00e-07dc-410e-ba3b-707f31701840"},"dd_version":"1.0.0","log_injection_enabled":false,"runtime_metrics_enabled":false,"profiling_enabled":false,"integrations_loaded":[]}
DATADOG TRACER DIAGNOSTIC - Agent Error: Network error trying to reach the agent: connect ECONNREFUSED 127.0.0.1:8126
✨  Done in 15.98s.
```

I believe that this is because I commited another mistake on the `index.d.ts` file which was to declare the `propagationBlocklist` property in the `HttpServer` interface, when it should have been in `HttpClient`. In the type test `propagationBlocklist` is declared on the client options, which treats it as an additional property and therefore the object is assignable to `HttpClient`.

I will proceed to fix both errors.